src/App/Fossa/Container/Sources/DockerArchive.hs

```diff
@@ -122,8 +122,9 @@ analyzeFromDockerArchive systemDepsOnly filters withoutDefaultFilters tarball =
   let imageBaseLayer = baseLayer image
       baseDigest = layerDigest imageBaseLayer
   osInfo <-
-    context "Retrieving OS Information" $
-      runTarballReadFSIO baseFs tarball getOsInfo
+    context "Retrieving OS Information"
+      . warnThenRecover @Text "Could not retrieve OS info"
```

Comment:
Should I log on stdout if we can't get system info? Warnings only print to the terminal if they use `--debug`.

Reply:
Hmm. I think this is good as is, but you're right that it would be better to log to stdout.

The log to stdout should be clear that "this is expected but we just wanted you to know in case it was confusing to you".

I was thinking we could link to the documentation, but the thing you'd like to is short enough that it's better to just include it in the message